### PR TITLE
Added script

### PR DESCRIPTION
**Pull request by GurgenBaghdasaryan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
-let reverse = (str) => {
-    let newString = "";
-
-    for (let i = str.length - 1; i >= 0; i--) {
-        newString += str[i];
-    };
-    return newString;
+function reverse (x,y){
```
Comment (arayik-99) [inline index.js]: Why is the function's name `reverse`?

File: index.js [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
+    let arr = new Array();
```
Comment (arayik-99) [inline index.js]: Couldn't this just be `let arr = []`?

File: index.js [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
+    for(let i = 0;i < y;i++){
```
Comment (arayik-99) [inline index.js]: Needs space after semicolons.

File: index.js (hunk 1 above)
Comment (GurgenBaghdasaryan) [reply index.js]: now fixed

**Pull request merged**